### PR TITLE
Update hds_record.rb

### PR DESCRIPTION
**Pull request by holmesie**
When converting the models, make sure the race patient characteristic uses the correct code system name (`CDC Race` is _incorrect_, the code system is `cdcrec`). When converting back, we need to remember that the old models used the incorrect code system name.

---

Pull requests into cqm-converter require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [x] This pull request describes why these changes were made.
- [x] Internal ticket for this PR: https://jira.mitre.org/secure/RapidBoard.jspa?rapidView=436&projectKey=BONNIE&view=planning&selectedIssue=BONNIE-1388
- [x] Internal ticket links to this PR
- [x] Code diff has been done and been reviewed
- [x] Tests are included and test edge cases
- [x] Tests have been run locally and pass
- [x] If applicable, the library version number in `cqm-converter.gemspec` has been updated

**Reviewer 1:**

Name: @okeefm 
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

**Reviewer 2:**

Name: @jbradl11 
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

### CODE REVIEW

File: lib/cqm/converter/hds_record.rb [hunk 1]
```diff
@@ -86,14 +87,16 @@ def to_qdm(record)
...
+        code = QDM::Code.new(race['code'], 'cdcrec', race['name'], '2.16.840.1.113883.6.238')
         patient.dataElements << QDM::PatientCharacteristicRace.new(dataElementCodes: [code])
       end
 
       # Convert patient characteristic sex.
       sex = record.gender
       if sex
-        code = QDM::Code.new(sex, 'AdministrativeSex', Utils.code_system_helper('AdministrativeSex'))
+        # See: https://phinvads.cdc.gov/vads/ViewCodeSystem.action?id=2.16.840.1.113883.5.1
+        code = QDM::Code.new(sex, 'AdministrativeGender', '2.16.840.1.113883.5.1')
```
Comment (okeefm) [inline lib/cqm/converter/hds_record.rb]: Does this call to `Code.new` need a `nil` between `AdministrativeGender` and the OID? Otherwise the OID will be treated as the `descriptor`.

File: lib/cqm/converter/qdm_patient.rb [hunk 2]
```diff
@@ -288,7 +288,7 @@ def unpack_patient_characteristics(patient, record)
       # Convert patient characteristic race.
       race = get_data_elements(patient, 'patient_characteristic', 'race').first
       race_code = race.dataElementCodes.first.symbolize_keys if race.dataElementCodes.any?
-      record.race = { code: race_code[:code], name: race_code[:descriptor], codeSystem: race_code[:codeSystem] } if race_code
+      record.race = { code: race_code[:code], name: race_code[:descriptor], codeSystem: 'CDC Race' } if race_code
```
Comment (okeefm) [inline lib/cqm/converter/qdm_patient.rb]: I assume we don't need to do anything in this file about the gender code, because the code coming in should already be `AdministrativeGender` and not `AdministrativeSex`?

File: lib/cqm/converter/hds_record.rb (hunk 1 above)
Comment (holmesie) [reply lib/cqm/converter/hds_record.rb]: Good catch! Fixed.
